public/js/gulliver.es6.js

```diff
@@ -107,24 +111,44 @@ class Gulliver {
     this._addRoute(/\/pwas\/(\d+)/, transitionStrategy, setupCharts, {
       showTabs: false,
       backlink: true,
-      subtitle: false
+      subtitle: true,
+      search: false
     });
 
-    const nop = () => {};
+    const nop = () => {
```

Comment:
This is not a `nop` anymore. Should change the method name to describe what it does.